src/components/tabs/helpers/tab-nesting-context.tsx

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, ReactNode } from 'react'
+
+type NestedTabContextValue = false
```

Comment:
🙋🏻 Should this be `boolean` instead of `false`? And should `useIsNested` return something like `Boolean(useContext(NestedTabContext))`?

🙋🏻 Also noticed that the `value` of the `TabNestingProvider` is hardcoded to `true`, does that mean this always returns `true` if the provider is rendered?